### ISSUE TITLE
Add version number to /healthcheck endpoint

### ISSUE THREAD

**Issue by pythoninja** [randomicu/fakedata-provider#111]
Example:
```
curl -s 127.0.0.1:8000/healthcheck

{
  "fakedata-backend": {
    "version": "1.0.7",
     "status": "UP"
  }
}
```

Refs:
1. https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package
2. https://github.com/tiangolo/fastapi/issues/702

**Comment by pythoninja** [comment]
Refs:
1. https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package
2. https://github.com/tiangolo/fastapi/issues/702

**Comment by pythoninja** [comment]
Closed in #164
